macOS_1_首次安装.sh

```
@@ -0,0 +1,249 @@
+        echo -e "${GREEN}[OK] 检测到本地 Conda: $MINICONDA_DIR${NC}"
+        # 尝试接受 TOS (如果存在)
+        "$MINICONDA_DIR/bin/conda" config --set channel_priority flexible 2>/dev/null || true
+        return 0
+    fi
+    
+    # 检查系统 Conda
+    if command -v conda &> /dev/null; then
+        SYSTEM_CONDA=$(which conda)
+        echo -e "${GREEN}[OK] 检测到系统 Conda: $SYSTEM_CONDA${NC}"
+        # 使用系统 Conda
+        MINICONDA_DIR=$(dirname $(dirname "$SYSTEM_CONDA"))
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)